doozer/tests/cli/test_gen_payload.py

```
@@ -993,6 +993,139 @@ async def test_apply_imagestream_update(self, mar_mock, binc_mock):
+                        tags=[
+                            dict(name="existing-tag"),
+                            dict(name="another-tag"),
+                        ]
+                    ),
+                )
+            ),
+        )
+        new_istags = [dict(name="existing-tag")]
+
+        # full_payload_tag_names=None means we can't determine deprecated tags
+        (pruning, adding) = await gpcli.apply_imagestream_update(istream_apiobj, new_istags, True, None)
```

Ruff (0.16.2)

[warning] Unpacked variable `adding` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)